docs/docs/edge-api/edge-api.info.mdx

```diff
@@ -0,0 +1,76 @@
+---
+id: edge-api
+title: "Edge API"
+description: ""
+sidebar_label: Introduction
+sidebar_position: 0
+hide_title: true
+custom_edit_url: null
+---
+
+import ApiLogo from "@theme/ApiLogo";
+import Heading from "@theme/Heading";
+import SchemaTabs from "@theme/SchemaTabs";
+import TabItem from "@theme/TabItem";
+import Export from "@theme/ApiExplorer/Export";
+
+<span
+  className={"theme-doc-version-badge badge badge--secondary"}
+  children={"Version: 1"}
+>
+</span>
+
+<Heading
+  as={"h1"}
+  className={"openapi__heading"}
+  children={"Edge API"}
+>
+</Heading>
+
+
+
+
+
+<div
+  style={{"marginBottom":"2rem"}}
+>
+  <Heading
+    id={"authentication"}
+    as={"h2"}
+    className={"openapi-tabs__heading"}
+    children={"Authentication"}
+  >
+  </Heading><SchemaTabs
+    className={"openapi-tabs__security-schemes"}
+  >
+    <TabItem
+      label={"API Key: EnvironmentKeyAuth"}
+      value={"EnvironmentKeyAuth"}
```

Comment:
Judging from site content we should change "EnvironmentKeyAuth" to a human-readable value, but I don't know exactly how to do it.

Reply:
From what I can tell, there's no real option to do this in the OpenAPI specification. See [here](https://swagger.io/docs/specification/v3_0/authentication/api-keys/).

The best we can do perhaps, is to just drop the `Auth` from the end? Like this: 

<img width="505" alt="image" src="https://github.com/user-attachments/assets/fb08f88a-051c-4784-a994-f66735f10ebb">

What do you think?

Reply:
That looks good, let's do it.

Reply:
Done